assets/templates/404.tmpl

```diff
@@ -0,0 +1,21 @@
+<div class="page-header slate">
```

Comment:
The dp-renderer library supplies an error page that I believe we normally use for displaying 404 pages: https://github.com/ONSdigital/dp-renderer/blob/main/assets/templates/error.tmpl

We also have a standard message for a 404 page I believe which really should be in the dp-renderer library, but it seems we pass it in instead. It might be worth considering adding this to the dp-renderer locale file for consistency. This may mean updating the error template to just have an easy way to say "this is a 404 error" and have the library assets take care of the rest. Regardless of whether you take this approach the wording can be found here: https://github.com/ONSdigital/dp-frontend-router/blob/ca043e4baa288361b6caeaa40680870fe834f909/middleware/serverError/handler.go#L30